sklearn/utils/testing.py

```diff
@@ -42,12 +42,21 @@
 except NameError:
     WindowsError = None
 
+from numpy.testing import assert_allclose
+from numpy.testing import assert_almost_equal
+from numpy.testing import assert_array_equal
+from numpy.testing import assert_array_almost_equal
+from numpy.testing import assert_array_less
+from numpy.testing import assert_approx_equal
```

Comment:
If you are ordering alphabetically, maybe you should put `from numpy.testing import assert_approx_equal` before of `from numpy.testing import assert_array_equal`